CHANGELOG.md

```diff
@@ -32,6 +32,7 @@
 
 ### Changes
 
+* [#4157](https://github.com/bbatsov/rubocop/issues/4553): Add `node_modules` to default excludes. ([@iainbeeston][])
```

Comment:
There's a mismatch between the leading number and the one in the URL (4157 vs. 4553) which is causing the build to fail. 🙂